ecosystem/sep-0031.md

```diff
@@ -376,7 +383,7 @@ the corresponding off-chain asset, after fees have been applied.
 | `fee_percent`         | number  | (optional) A percentage fee in percentage points. Leave blank if there is no fee or fee calculation cannot be modeled using a fixed and percentage fee.                                                                                                 |
 | `sender_sep12_type`   | string  | (**deprecated**, optional) The value of the `type` parameter the Sending Anchor should use for a `SEP-12 GET /customer` request. This field can be omitted if no KYC is necessary. Use a value from `sep12.sender.types` instead if any are present.    |
 | `receiver_sep12_type` | string  | (**deprecated**, optional) The value of the `type` parameter the Sending Anchor should use for a `SEP-12 GET /customer` request. This field can be omitted if no KYC is necessary. Use a values from `sep12.receiver.types` instead if any are present. |
-| `fields`              | object  | (**deprecated**, optional) An object containing the per-transaction parameters required in `POST /transactions` requests. Pass [SEP-9] fields via [SEP-12 PUT /customer](sep-0012.md#customer-put) instead.                                             |
+| `fields`              | object  | (**deprecated**, optional) An object that contains multiple key-value pairs, where each key represents a field related to the transaction, and the value is an `transaction` object describing the field's details.                                     |
```

Comment:
Since we are adding `type` to `fields`, I think we should remove `deprecated`.

Reply:
Neither of these descriptions quite make sense. The original version of the `fields` object schema looked like this:

```js
{
  "fields": {
    "transaction": {}
  }
}
```

We deprecated `fields` because its only purpose was for `transaction` to specify request parameters required to be sent in `POST /transactions` requests.

Now that we're adding `type`, I think we have a couple options:

- Deprecate `fields.transaction` specifically, and un-deprecated `fields`. This way we can add the `type` key to the `fields` object:

```js
{
  "fields": {
    "type": {
      "description": "",
      "choices": [],
      "optional"
    }
  }
}
```

- Un-deprecate both `fields` and `field.transaction`, and add `type` as a subfield of `field.transactions`:

```js
{
  "fields": {
    "transaction": {
      "type": {
        "description": "",
        "choices": [],
        "optional"
      }
    }
  }
}
```